src/facade/files/__name@dasherize__/__name@dasherize__.facade.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Store } from '@ngrx/store';<% if (store) { %>
+import { <%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageSelectors } from './shared/store';<% } %><% if (isNgrxInstalled) { %>
```

Comment:
@Goodmain we can shorten this `classify` call:

```ts
classify(section, parent, name)
```

Reply:
@dmitryusenko Fixed